libraries/enigma.h

```diff
@@ -182,6 +182,29 @@ namespace eng
 			
 			return output;
 		}
+		
+		char transcribe(char __letter, int __disk_number) // this function changes input letter according to connections in the disk
+		{
+			char output = enigma.disks[__disk_number].foward(__char_to_number__(__letter));
```

Comment:
Jak używamy atrybutów klasy to nie trzeba pisać nazwy klasy przed. Można sie do nich odwoływać bezpośrednio jak do zmiennych. W tym przypadku `... = disks[...`

Reply:
a no racja , bo zastanawiałem się czy bedziemy to robić w głównej czy tu i zdecydowałem że tu chyba sensowniej